src/tests/device.test.js

```diff
@@ -0,0 +1,21 @@
+import supertest from 'supertest';
+import app from '../http/app';
+
+const request = supertest(app);
+it('gets the version endpoint', async (done) => {
```

Comment:
The test for the version endpoint has already been written in a separate file so you can remove this.